### PR TITLE
feat: Completeted typography

### PR DESCRIPTION
**Pull request by Vishruta-Patil**

### CODE REVIEW

**Review by Devansu-Yadav — APPROVED** [review]
Looks good!
Preview Link to the component - https://deploy-preview-8--magnizent-ui.netlify.app/components/topography/topography

File: Components/Topography/topography.css [hunk 1]
```diff
@@ -0,0 +1,59 @@
+.txt {
```
Comment (Devansu-Yadav) [inline Components/Topography/topography.css:1]: Rename the Component folder as well as the file to `Typography` and `typography` respectively.

File: Components/Topography/topography.html [hunk 2]
```diff
@@ -0,0 +1,290 @@
+<!DOCTYPE html>
```
Comment (Devansu-Yadav) [inline Components/Topography/topography.html:1]: Same here. Rename the file to `typography`

File: Components/Topography/topography.html [hunk 3]
```diff
@@ -0,0 +1,290 @@
...
+            <a class="drawer-links" href="../Card/card.html">Card</a>
+          </p>
+          <p class="drawer-inner-header">
+            <a class="drawer-links" href="../Image/image.html">Image</a>
+          </p>
+          <p class="drawer-inner-header">
+            <a class="drawer-links" href="../Input/input.html">Input</a>
+          </p>
+          <p class="drawer-inner-header">
+            <a class="drawer-links" href="../Topography/topography.html"
```
Comment (Devansu-Yadav) [inline Components/Topography/topography.html:108]: Same here. Make the relevant change here as well.

File: Components/Topography/topography.html [hunk 4]
```diff
@@ -0,0 +1,290 @@
...
+      href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="../../style/header.css" />
+    <link rel="stylesheet" href="../../style/menu.css" />
+    <link rel="stylesheet" href="../../style/mainContent.css" />
+    <link rel="stylesheet" href="../../style/codeSnippet.css" />
+    <link rel="stylesheet" href="../../style/footer.css" />
+    <link rel="stylesheet" href="../../style/sidecontent.css" />
+    <link rel="stylesheet" href="topography.css" />
```
Comment (Devansu-Yadav) [inline Components/Topography/topography.html:24]: Same here. Rename the CSS file and update the stylesheet import here.

File: Components/Input/input.html [hunk 5]
```diff
@@ -106,6 +105,11 @@ <h3 class="drawer-header">Get Started</h3>
           <p class="drawer-inner-header">
             <a class="drawer-links" href="../Input/input.html">Input</a>
           </p>
+          <p class="drawer-inner-header">
+            <a class="drawer-links" href="../Topography/topography.html"
```
Comment (Devansu-Yadav) [inline Components/Input/input.html:109]: Same here. Update the file name.

File: Components/Card/card.html [hunk 6]
```diff
@@ -91,6 +91,9 @@ <h3 class="drawer-header">Get Started</h3>
                 <p class="drawer-inner-header">
                     <a class="drawer-links" href="../Input/input.html">Input</a>
                 </p>
+                <p class="drawer-inner-header">
+                    <a class="drawer-links" href="../Topography/topography.html">Typography</a>
```
Comment (Devansu-Yadav) [inline Components/Card/card.html:95]: Same here. Update the file name

File: Components/Button/button.html [hunk 7]
```diff
@@ -91,6 +91,9 @@ <h3 class="drawer-header">Get Started</h3>
         <p class="drawer-inner-header">
           <a class="drawer-links" href="../Input/input.html">Input</a>
         </p>
+        <p class="drawer-inner-header">
+          <a class="drawer-links" href="../Topography/topography.html">Typography</a>
```
Comment (Devansu-Yadav) [inline Components/Button/button.html:95]: Same thing here. Rename and update the file name.

File: Components/Badge/badge.html [hunk 8]
```diff
@@ -93,6 +93,9 @@ <h3 class="drawer-header">Get Started</h3>
           <p class="drawer-inner-header"><a class="drawer-links" href="../Card/card.html">Card</a></p>
           <p class="drawer-inner-header"><a class="drawer-links" href="../Image/image.html">Image</a></p> 
           <p class="drawer-inner-header"><a class="drawer-links" href="../Input/input.html">Input</a></p>
+          <p class="drawer-inner-header">
+            <a class="drawer-links" href="../Topography/topography.html">Typography</a>
```
Comment (Devansu-Yadav) [inline Components/Badge/badge.html:97]: Same thing here. Rename and update the file name.

File: Components/Avatar/avatar.html [hunk 9]
```diff
@@ -85,13 +85,32 @@ <h3 class="drawer-header">Get Started</h3>
...
+            <a class="drawer-links" href="../Card/card.html">Card</a>
+          </p>
+          <p class="drawer-inner-header">
+            <a class="drawer-links" href="../Image/image.html">Image</a>
+          </p>
+          <p class="drawer-inner-header">
+            <a class="drawer-links" href="../Input/input.html">Input</a>
+          </p>
+          <p class="drawer-inner-header">
+            <a class="drawer-links" href="../Topography/topography.html"
```
Comment (Devansu-Yadav) [inline Components/Avatar/avatar.html:110]: Same thing here. Rename and update the file name.

File: Components/Alert/alert.html [hunk 10]
```diff
@@ -85,13 +84,32 @@ <h3 class="drawer-header">Get Started</h3>
...
+            <a class="drawer-links" href="../Card/card.html">Card</a>
+          </p>
+          <p class="drawer-inner-header">
+            <a class="drawer-links" href="../Image/image.html">Image</a>
+          </p>
+          <p class="drawer-inner-header">
+            <a class="drawer-links" href="../Input/input.html">Input</a>
+          </p>
+          <p class="drawer-inner-header">
+            <a class="drawer-links" href="../Topography/topography.html"
```
Comment (Devansu-Yadav) [inline Components/Alert/alert.html:109]: Same thing here. Rename and update the file name.

**Review by Devansu-Yadav — APPROVED** [review]

**Review by nerdyspook — APPROVED** [review]
The code looks good, you can merge.

**Pull request merged**